src/main/java/net/joelinrome/plasmacraft/compat/DeuteriumExtractorRecipies.java

```diff
@@ -0,0 +1,60 @@
+package net.joelinrome.plasmacraft.compat;
+
+import mezz.jei.api.constants.VanillaTypes;
+import mezz.jei.api.gui.builder.IRecipeLayoutBuilder;
+import mezz.jei.api.gui.drawable.IDrawable;
+import mezz.jei.api.helpers.IGuiHelper;
+import mezz.jei.api.recipe.IFocusGroup;
+import mezz.jei.api.recipe.RecipeIngredientRole;
+import mezz.jei.api.recipe.RecipeType;
+import mezz.jei.api.recipe.category.IRecipeCategory;
+import net.joelinrome.plasmacraft.PlasmaCraft;
+import net.joelinrome.plasmacraft.block.ModBlocks;
+import net.joelinrome.plasmacraft.repice.DeuteriumExtractorRecipe;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+
+public class DeuteriumExtractorRecipies implements IRecipeCategory<DeuteriumExtractorRecipe> {
+    public static final ResourceLocation UID = new ResourceLocation(PlasmaCraft.MOD_ID, "deuterium_extracting");
+    public static final ResourceLocation TEXTURE = new ResourceLocation(PlasmaCraft.MOD_ID,
+             "textures/gui/deuterium_exractor_gui.png");
+
+    public static final RecipeType<DeuteriumExtractorRecipe> DEUTERIUM_EXTRACTOR_TYPE =
+            new RecipeType<>(UID, DeuteriumExtractorRecipe.class);
+
+    private final IDrawable background;
+    private final IDrawable icon;
+
+    public DeuteriumExtractorRecipies(IGuiHelper helper) {
+        this.background = helper.createDrawable(TEXTURE, 0, 0, 176, 85);
+        this.icon = helper.createDrawableIngredient(VanillaTypes.ITEM_STACK, new ItemStack(ModBlocks.DEUTERIUM_EXTRACTOR_BLOCK.get()));
+    }
+
+    @Override
+    public RecipeType<DeuteriumExtractorRecipe> getRecipeType() {
+        return DEUTERIUM_EXTRACTOR_TYPE;
+    }
+
+    @Override
+    public Component getTitle() {
+        return Component.literal("Deuterium Extractor");
```

Comment:
Could replace this line with the following which references to the key from `src/main/resources/assets/plasmacraft/lang/en_us.json`. This just means that now we would only need to change the block name once from this file instead of looking for all the `Deuterium Extractor` references
```java
Component.translatable("block.plasmacraft.deuterium_extractor")
```